relationshiptypes/radon.relationships.monitoring/GCPIsMonitoredBy/RelationshipType.tosca

```diff
@@ -0,0 +1,38 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+relationship_types:
+  radon.relationships.monitoring.GCPIsMonitoredBy:
+    derived_from: tosca.relationships.ConnectsTo
+    metadata:
+      targetNamespace: "radon.relationships"
+      abstract: "false"
+      final: "false"
+    interfaces:
+      Configure:
+        type: tosca.interfaces.relationship.Configure
+        inputs:
+          monitored_function:
+            type: string
+            required: true
+            default: { get_property: [ SOURCE, name ] }
+          pushgateway_ip:
+            type: string
+            required: true
+            default: { get_attribute: [ TARGET, pushgateway_ip ] }
+          user_email:
+            type: string
+            required: true
+            default: { get_property: [ TARGET, user_email ] }
+          grafana_api_ip: 
+            type: string
+            required: true
+            default: { get_property: [ TARGET, grafana_api_ip ] }
+          service_account_file:
+            type: string
+            required: true
+            default: "/tmp/service_account.json"
```

Comment:
I remember that we had issues with using `get_input` here, but I'm wondering if we can get this info from the related `SOURCE` or `TARGET` nodes?!

Reply:
Unfortunately not. As I remember, both SOURCE and/or TARGET get this value through a dependency. We could modify the target or source tosca specification but then again the applicability will not be general.

We opened an issue about this topic and I guess it will be resolved at some point. So for the moment we could live the assigned hard coded value and then revisit the issue. If this causes issues though we should find another way.

Reply:
I see. But what about `get_property: [TARGET, dependency, service_account_file ]`?

We can merge it as is I think, but this could be worth testing.